variables.tf

```diff
@@ -254,10 +254,22 @@ variable "logs_config" {
   description = "Configuration for the builds to store log data to CloudWatch or S3."
 }
 
+variable "default_permissions_enabled" {
```

Comment:
Could we use the presence of the custom policy instead of this flag?

Reply:
it's possible one wants the defaults and custom policy. I used the enable variable to follow conventions elsewhere in the module.

happy to change that, if you'd prefer it to be an either or.